core/src/main/scala/org/http4s/package.scala

```diff
@@ -77,4 +78,7 @@ package object http4s { // scalastyle:ignore
   type Http4sSyntax = syntax.AllSyntax
   @deprecated("Moved to org.http4s.syntax.all", "0.16")
   val Http4sSyntax = syntax.all
+
+  def httpRoutesToApp[F[_]: ConcurrentEffect](httpRoutes: HttpRoutes[F]): HttpApp[F] =
+    HttpApp[F](req => httpRoutes(req).getOrElse[Response[F]](Response.notFound[F]))
```

Comment:
Actually, we already have a syntax for this. With `import org.http4s.syntax.kleisli._` we can do `httpRoutes.orNotFound`. So I don't think we need this as a separate function.